test/test_env.py

```diff
@@ -992,6 +992,24 @@ def test_steptensordict(
         assert out is next_tensordict
 
 
+@pytest.mark.skipif(not _has_gym, reason="no gym")
+@pytest.mark.parametrize("env_name", ["Pendulum-v1", "ALE/Pong-v5"])
+@pytest.mark.parametrize("frame_skip", [1, 4])
+def test_batch_lock_gym(env_name, frame_skip, seed=0):
+    env = GymEnv(env_name, frame_skip=frame_skip)
+    assert env.batch_locked
+
+    with pytest.raises(RuntimeError):
```

Comment:
can we check the reason
I believe it's `raises(RuntimeError, reason="error message")`

Reply:
pytest doesn't support reason arg for raises. The only other arg supported is match to do a regex on the raised error message, is this what you expected